Database/src/main/resources/changelog1.xml

```diff
@@ -781,4 +781,36 @@
         </insert>
     </changeSet>
 
+    <changeSet author="CAF" id="1512411132000-1">
+        <insert tableName="tbl_policy_type">
+            <column name="item_name" value="Document Policy Type"/>
```

Comment:
I don't know if we really want to add this as it doesn't support chaining.

Reply:
Removed. Composite Document type also removed in favor of Chained Action.